tests/testthat/test-AbstractGraphReporter.R

```diff
@@ -0,0 +1,65 @@
+context("Abstract Graph Reporter Tests")
+
+##### TEST SET UP #####
+
+rm(list = ls())
+# Configure logger (suppress all logs in testing)
+loggerOptions <- futile.logger::logger.options()
+if (!identical(loggerOptions, list())){
+  origLogThreshold <- loggerOptions[[1]][['threshold']]
+} else {
+  origLogThreshold <- futile.logger::INFO
+}
+futile.logger::flog.threshold(0)
+
+##### TESTS #####
+
+## Structure Available ##
+
+test_that('AbstractGraphReporter structure is as expected', {
+  
+  expect_named(object = AbstractGraphReporter$public_methods
+               , expected = c(
+                 "calculate_network_measures",
+                 "get_summary_view",
+                 "set_graph_layout",
+                 "plot_network",
+                 "set_plot_node_color_scheme",
+                 "get_plot_node_color_scheme",
+                 "clone"
+               )
```

Comment:
please add these two arguments:

```
ignore.order = TRUE, ignore.case = FALSE
```

The order has 0 impact on the execution of this code...would be frustrating to have this test fail if we reshuffled the ordering inside the class or if some obscure detail of R6 changed and the `$public_methods` implementation changed

Also TIL `$public_methods`. Nice! Really like this

Reply:
this same comment about `ignore.*` applies for all your uses of `expect_named`, please

Reply:
good call.  I agree for all R6 objects.